MaxText/inference_mlperf/README.md

```diff
@@ -14,13 +14,22 @@ source .env/bin/activate
 
 ### Install loadgen
 ```
-sudo apt-get install python3-dev
-sudo apt-get install build-essential -y
-git clone git@github.com:mlcommons/inference.git
-cd inference/
-cd loadgen/ && python3 -m pip install .
+pip install absl-py numpy
+git clone --recurse-submodules https://github.com/mlcommons/inference.git mlperf_inference
+cd mlperf_inference/loadgen
+CFLAGS="-std=c++14 -O3" python -m pip install .
 ```
 
+If you run into an issue like the following:
+
+```
+ImportError: /mnt/disks/jacobplatin/anaconda3/envs/llama4-maxtext/bin/../lib/libstdc++.so.6: version `GLIBCXX_3.4.30'
+not found (required by /mnt/disks/jacobplatin/anaconda3/envs/llama4-maxtext/lib/python3.10/site-packages/mlperf_loadgen.cpython-310-x86_64-linux-gnu.so)
+```
+
+Please try running `conda install -c conda-forge gcc_linux-64 gxx_linux-64 libstdcxx-ng` and reinstalling `loadgen`
```

Comment:
Nit: Not sure if we should add conda into the mix here or not since it's not mentioned anywhere else. Maybe make the fix slightly more agnostic (if possible). 

At the very least we can make it a bit easier to parse by simplifying the dirs: 
```
venv/lib/libstdc++.so.6
venv/lib/python3.10/site-packages/mlperf_loadgen.cpython-310-x86_64-linux-gnu.so
```